certified-connectors/Revelation helpdesk/apiProperties.json

```diff
@@ -0,0 +1,68 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "revSiteHost": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "displayName": "Revelation URL",
+          "description": "Your Revelation site URL (Example: https://support.yourdomain.com)",
+          "tooltip": "Provide your Revelation helpdesk site URL. This must be a secure Url (https://) and must NOT end with '/'"
+        }
+      },
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Revelation.PowerAutomate",
+          "scopes": [
+            "openid profile Revelation.Api"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://revelationauth.azurewebsites.net/common/connect/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://revelationauth.azurewebsites.net/common/connect/token"
+            },
+            "refreshUrl": {
+              "value": "https://revelationauth.azurewebsites.net/common/connect/token"
+            }
+          }
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Login with your Revelation helpdesk credentials",
+          "displayName": "Login with your credentials",
+          "tooltip": "Login with your credentials"
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@MVliegenthart You have agreed not to have this iconBrandColor for your connector. 

![image](https://user-images.githubusercontent.com/56276394/154598196-c0366e7f-8e64-41bc-9811-b6336b207447.png)

Reply:
@sriyen-msft Apologies for the oversight, this has been fixed.